packages/calypso-e2e/src/lib/pages/themes-customizer-page.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import { BaseContainer } from '../base-container';
+
+/**
+ * Type dependencies
+ */
+import { Frame } from 'playwright';
+
+const selectors = {
+	frame: 'iframe[title="Customizer"]',
+};
+
+/**
+ * Component representing the Appearance > Customize page.
+ *
+ * @augments {BaseContainer}
+ */
+export class ThemesCustomizerPage extends BaseContainer {
+	frame!: Frame;
```

Comment:
Since we can now with Typescript, should this be a private class member? I imagine this variable being used by various methods in this class, but probably not as much by calling classes. Do you think it will be needed by the consuming tests?

Reply:
You are right - these are internal class member and do not need to be accessed externally. It would make sense to have these members (`frame`, `page`, `block`, etc.) be private in addition to the `_postInit`, `_preInit` and similar methods.

I will start with making the members in this class private first.

Reply:
Class no longer exists, so just resolving this comment!